angular/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "build": "npm run set-version && ng build spark-angular",
-    "test": "jest --config ./projects/spark-angular/jest.config.js",
+    "test": "jest --config ./projects/spark-angular/jest.config.js --verbose=false",
```

Comment:
Adding in the verbose flag here makes it so console.logs show up. Otherwise, they don't.

Reply:
dont need the =false